lib/mix/nerves/utils.ex

```diff
@@ -160,33 +160,54 @@ defmodule Mix.Nerves.Utils do
     |> String.replace("::", ":")
   end
 
-  @spec parse_version(String.t()) :: {:error, String.t()} | {:ok, Version.t()}
-  def parse_version(vsn) do
-    cond do
-      # Strict semver
-      Regex.match?(
-        ~r/^((([0-9]+)\.([0-9]+)\.([0-9]+)(?:-([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)(?:\+([0-9a-zA-Z-]+(?:\.[0-9a-zA-Z-]+)*))?)$/,
-        vsn
-      ) ->
-        {:ok, Version.parse!(vsn)}
-
-      # x.x
-      Regex.match?(~r/^([0-9]+)\.([0-9]+)$/, vsn) ->
-        {:ok, Version.parse!(vsn <> ".0")}
-
-      # x.x.x.x
-      Regex.match?(~r/^([0-9]+)\.([0-9]+)\.([0-9]+)\.([0-9]+)$/, vsn) ->
-        [major, minor, patch | _tail] = String.split(vsn, ".")
-
-        vsn = Enum.join([major, minor, patch], ".")
-        {:ok, Version.parse!(vsn)}
-
-      # unknown
-      true ->
-        {:error, "Unable to Version.parse #{inspect(vsn)}"}
+  @doc """
+  Parse OTP versions
+
+  OTP versions can have anywhere from 2 to 5 parts. Normalize this into
+  a 3-part version for convenience. This is a lossy operation, but it
+  doesn't matter the the checks needed in this project.
```

Comment:
```suggestion
  doesn't matter because the checks aren't needed in this project.
```
I think this is what you meant?